rna-transcription/rna-transcription.js

```diff
@@ -0,0 +1,24 @@
+function DnaTranscriber () {};
+
+DnaTranscriber.prototype.toRna = function(dnaStrand) {
+
+    var dnaToRna = new Map();
+
+    dnaToRna.set('A', 'U');
+    dnaToRna.set('T', 'A');
+    dnaToRna.set('C', 'G');
+    dnaToRna.set('G', 'C');
+
+    this.rnaStrand = new String;
```

Comment:
It looks like there's no point in `this.rnaStrand` being instance variable because it's always returned and `DnaTranscriber` is "general". It doesn't need to remember its output. Hint: [reduce](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce)

```
    var dnaToRna = new Map();
 
    dnaToRna.set('A', 'U');
    dnaToRna.set('T', 'A');
    dnaToRna.set('C', 'G');
    dnaToRna.set('G', 'C');
```

Is ran on every `toRna` execution => try to optimize this part ;)